moped-database/views/component_arcgis_online_view.sql

```diff
@@ -182,6 +182,7 @@ SELECT
     plv.project_partners,
     plv.task_order_names,
     plv.funding_source_name,
+    plv.funding_source_and_program_names,
```

Comment:
@frankhereford I like that this is named `funding_source_and_program_names` in the project list view, but could you alias this as `funding_sources` here? That is the name that was added to the AGOL dataset and also aligns with the request from ATSD. 🙏